algoliasearch/src/main/java/com/algolia/search/saas/BaseIndex.java

```diff
@@ -310,6 +310,7 @@ protected JSONObject deleteObjects(List<String> objects) throws AlgoliaException
     protected void deleteByQuery(Query query) throws AlgoliaException {
         query.setAttributesToRetrieve("objectID");
         query.setHitsPerPage(100);
+        query.enableDistinct(false);
```

Comment:
@ElPicador: Sure, doing so!
@clement-leprovost-algolia: Thanks, I'll correct that as well.